packages/node_modules/@ciscospark/widget-space/src/enhancers/errors.js

```diff
@@ -94,7 +94,9 @@ function checkForErrors(props) {
   }
 
   const invalidActivityId = 'ciscospark.container.space.error.invalidActivity';
-  const validActivity = initialActivity ? activityTypes.some((activity) => activity.name === initialActivity) : true;
+  const defaultActivity = 'message';
```

Comment:
this change doesn't really fit into the commit message or PR. intentional?

Reply:
whoops, i shouldve split them. it's a bug fix i noticed because the space widget demo doesn't set an initial activity, so i just fixed the bug to check for the default activity instead of adding an initial activity to the space demo.